docs/setup_installation/admin/monitoring/export-metrics.md

```diff
@@ -2,73 +2,126 @@
+
+```bash
+export NAMESPACE=hopsworks
+kubectl -n $NAMESPACE get svc prometheus-external -ojsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+!!!Warning
+    It will take a few seconds until an IP address is assigned to the Service
+
+We will use this IP address in Step 2
+
+#### Step 2
+Edit the configuration file of **Prometheus B** server and append the following Job under `scrape_configs`
 
 !!! note
-    Replace IP_ADDRESS with the actual address of Hopsworks server
+    Replace IP_ADDRESS with the IP address from Step 1 or the IP address of Prometheus service if it is directly accessible.
+    The snippet below assumes Hopsworks services runs at Namespace **hopsworks**
 
 ```yaml
 - job_name: 'federate'
-    scrape_interval: 15s
+  scrape_interval: 15s
 
-    honor_labels: true
-    metrics_path: '/federate'
+  honor_labels: true
+  metrics_path: '/federate'
 
-    params:
-      'match[]':
-        - '{job="airflow"}'
-        - '{job="pushgateway"}'
-        - '{job="hadoop"}'
-        - '{job="hopsworks"}'
+  params:
+    'match[]':
+      - '{namespace="hopsworks"}'
 
-    static_configs:
-      - targets:
-        - 'IP_ADDRESS:9089'
+  static_configs:
+    - targets:
+      - 'IP_ADDRESS:9090'
 ```
 
-These are the basic labels gathered by Hopsworks.
-
-* If your Hopsworks cluster runs **without** Kubernetes append `'{job="cadvisor"}'` to `match[]` list
+The configuration above will scrape for services metrics under the *hopsworks* Namespace. If you want to additionally
+scrape *user application* metrics then append `'{job="pushgateway"}'` to the matchers, for example:
 
-* If your Hopsworks cluster runs **with** Kubernetes append the following labels to `match[]`
-    * `'{job=~"knative.+"}'`
-    * `'{job="kubernetes-cadvisor"}'`
-    * `'{job="istio-envoy"}'`
-    * `'{job="kube-state-metrics"}'`
-    * `'{job="cadvisor"}'`
-    * `'{job="cadvisor"}'`
-    * `'{job="cadvisor"}'`
+```yaml
+  params:
+    'match[]':
+      - '{namespace="hopsworks"}'
+      - '{job="pushgateway"}'
+```
 
-#### Step 3
-Finally restart Prometheus service with `sudo systemctl restart prometheus`
+Depending on the Prometheus setup you might need to restart **Prometheus B** service to pick up the new configuration.
+For more details on federation visit Prometheus [documentation](https://prometheus.io/docs/prometheus/latest/federation/#cross-service-federation)
 
 ### Custom service
 Prometheus can push metrics to another custom resource via HTTP. The custom service is responsible for handling the received metrics.
 To push metrics with this method we use the `remote_write` configuration.
 
-
 We will only give a sample configuration as `remote_write` is extensively documented in Prometheus [documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)
 In the example below we push metrics to a custom service listening on port 9096 which transforms the metrics and forwards them.
 
+#### Step 1
```

Comment:
Is this really a good idea? The config map will be overwritten with the next `helm upgrade`.

Looks like it's possible to configure the `remoteWrite` from the helm repo:
https://github.com/prometheus-community/helm-charts/blob/fe79476450db94f79d65c4863d63484c26a12037/charts/prometheus/values.yaml#L280C3-L280C14

I think we should change the below intructions to tell users to add this configuration in the values.yaml file.

Reply:
True, that's better. I'll change the instructions